lib/Zonemaster/Backend/DB.pm

```diff
@@ -112,6 +112,34 @@ sub get_batch_job_result {
 	return \%result;
 }
 
+sub process_unfinished_tests {
+    my ( $self ) = @_;
+    
+    my $dbh = $self->dbh;
```

Comment:
Would it be possible to move the `dbh`, `prepare`, `execute` and `do` calls to the database adapters? That way we could write tests for the business logic.

Reply:
I'm not sure to understand what you mean here.

Reply:
What I'm trying to get at is that I'd like to have a unit test that demonstrates that the garbage collection works. But I realize this requires a larger refactoring. I'll create a separate issue for that purpose.